tools/systematic.py

```diff
@@ -255,6 +382,7 @@ def scaleTopMassSystematic( upper_uncertainty, lower_uncertainty, topMasses, top
 
 def get_sign(central, upper, lower, upper_variation, lower_variation):
     '''
+    Currently Obsolete.
```

Comment:
can you add a `@deprecated` decorator? This way python will produce a warning if the function is used somewhere.